src/App.tsx

```diff
@@ -1,16 +1,21 @@
+import { useState } from 'react';
+import { Movie } from './types/Movie';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
 
 export const App = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+  const onAdd = (movie : Movie) => setMovies(state => [...state, movie]);
```

Comment:
```suggestion
  const handleMovieAdding = (movie : Movie) => setMovies(state => [...state, movie]);
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364